modules/ROOT/pages/get-started.adoc

```diff
@@ -4,9 +4,13 @@ After you become familiar with the API Governance UI and concepts covered in the
 
 == Before You Begin
 
-To access the API Governance console:
+* To access the API Governance console, you must have the *API Governance* > *Governance Administrator* permission.
 
-* You must have the *Anypoint API Governance administrator* permission set by your Anypoint Platform administrator.
+* Opt in to the new features using the prompts in the API Governance UI. This enables full instance governance for API versions and a detailed governance report in API Governance and API Manager. See the xref:release-notes::api-governance.adoc[API Governance Release Notes] for details.
```

Comment:
```suggestion
* Opt in to the new features using the prompts in the API Governance UI. This opt-in enables full-instance governance for API versions and a detailed governance report in API Governance and API Manager. See the xref:release-notes::api-governance.adoc[API Governance Release Notes] for details.
```

Reply:
(need a qualifier for "this" - maybe it's "option"?

Reply:
This edit doesn't quite work. I'll look more closely at it to accept the spirit of the edit.